src/containers/shared/css/global.scss

```diff
@@ -14,30 +14,7 @@
     url('../fonts/currency_symbols.ttf') format('truetype'),
     url('../fonts/currency_symbols.woff') format('woff'),
     url('../fonts/currency_symbols.svg') format('svg');
-}
-
-@font-face {
-  font-family: 'Space Mono';
-  src: url('../fonts/SpaceMono-Regular.ttf');
-}
-
-@font-face {
-  font-family: 'Space Mono';
-  font-style: italic;
-  src: url('../fonts/SpaceMono-Italic.ttf');
-}
-
-@font-face {
-  font-family: 'Space Mono';
-  font-weight: bold;
-  src: url('../fonts/SpaceMono-Bold.ttf');
-}
-
-@font-face {
-  font-family: 'Space Mono';
-  font-style: italic;
-  font-weight: bold;
-  src: url('../fonts/SpaceMono-BoldItalic.ttf');
+  unicode-range: u+e900-e901;
```

Comment:
indentation looks off

Reply:
Optical illusion. https://github.com/ripple/explorer/blob/f07e7f39753ed245150643a4a21342fb3f6413a8/src/containers/shared/css/global.scss#L17

Reply:
add a comment in the CSS

Reply:
also remove eot files

Reply:
consider changing `font-family` to something super descriptive like `crypto_symbols`